twine/commands/check.py

```diff
@@ -170,18 +170,19 @@ def check(
 def main(args: List[str]) -> bool:
     """Execute the ``check`` command.
 
-    This currently only validates ``long_description``, but more checks could be
-    added.
-
-    Note: check is not intended to catch all reasons why an upload to PyPI might fail.
-
     :param args:
         The command-line arguments.
 
     :return:
         The exit status of the ``check`` command.
     """
-    parser = argparse.ArgumentParser(prog="twine check")
+    description = """
+    This currently only validates ``long_description``, but more checks could be
+    added.
+
+    Note: check is not intended to catch all reasons why an upload to PyPI might fail.
+    """
```

Comment:
```suggestion
    description = """This currently only validates if long_description will render on PyPI.
    
    ** Caution: This does **not** verify everything that PyPI will.

    ** Caution: This **never** verifies anything pertinent to non-PyPI index behaviour.
    
    By virtue of performing rendering checks on the long_description, this may validate
    other metadata characteristics loosely. 
    """
```